packages/instrumentation-socket.io/README.md

```diff
@@ -36,6 +36,14 @@ socket.io instrumentation has few options available to choose from. You can set
 | `emitHook` | `SocketIoHookFunction` | hook for adding custom attributes before socket.io emits the event |
 | `onHook` | `SocketIoHookFunction` | hook for adding custom attributes before the event listener (callback) is invoked |
 | `traceReserved` | `boolean` | set to true if you want to trace socket.io reserved events (see https://socket.io/docs/v4/emit-cheatsheet/#Reserved-events) |
+| `filterHttpTransport`| `HttpTransportInstrumentationConfig` | set if you want to filter out the HTTP traces when using HTTP polling as the transport (see options below)
```

Comment:
What about suppressing the http span on outgoing traffic?

Reply:
The socket.io client pulls the data so there are no outgoing HTTP requests

Reply:
Right, good point :)